### PR TITLE
feat(Box): make ShadowOffset customizable

### PR DESCRIPTION
**Pull request by Kooshaba**
### PR Checklist

- [x] Related to designs:
- [x] Related to JIRA ticket: [LX-4327](https://codecademy.atlassian.net/browse/LX-4327)
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@4.2.11-alpha.437e56.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f89de880e2c910e2cd41159--gamut-preview.netlify.app>https://5f89de880e2c910e2cd41159--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f89de880e2c910e2cd41159>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@4.3.3-alpha.531c3f.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f89e0c783cc5713171c8d78--gamut-preview.netlify.app>https://5f89e0c783cc5713171c8d78--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f89e0c783cc5713171c8d78>Deploy Logs</a></p>

File: packages/gamut-labs/src/experimental/Box/utils.ts [hunk 1]
```diff
@@ -1,20 +1,13 @@
...
 
 const shadowOffset = 2;
 
 export const createShadowOffset = (
-  offset: PaddingSizes,
-  direction: 'left' | 'right'
+  offset: ShadowOffset,
 ) => {
-  const [xCoeff, yCoeff] = offsets[direction];
+  const [xCoeff, yCoeff] = [1, -1];
```
Comment (JoshuaKGoldberg) [inline packages/gamut-labs/src/experimental/Box/utils.ts:10]: These can be removed now and the constants directly applied in the usage, yes?

**Review by JoshuaKGoldberg — APPROVED** [review]
LGTM, with the coefficients caveat 🚀

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@4.3.4-alpha.904c02.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f8d998b0b10bf5ebe61e2bd--gamut-preview.netlify.app>https://5f8d998b0b10bf5ebe61e2bd--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f8d998b0b10bf5ebe61e2bd>Deploy Logs</a></p>

**Review by codecaaron — APPROVED** [review]
@Kooshaba I'd re run prettier locally, and push up the result, then this should be good to go :)

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@4.3.4-alpha.b245a8.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f8dafef8d956d7dd7f71716--gamut-preview.netlify.app>https://5f8dafef8d956d7dd7f71716--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f8dafef8d956d7dd7f71716>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@4.3.4-alpha.da9029.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f8db1f7dafb1273662c6f74--gamut-preview.netlify.app>https://5f8db1f7dafb1273662c6f74--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f8db1f7dafb1273662c6f74>Deploy Logs</a></p>

**Pull request merged**